### DISCUSSION TITLE
Stand-alone Flet mobile app?

### DISCUSSION THREAD

**Discussion by echogit** [flet-dev/flet#1822] [General]
From what I've seen [on this page](https://flet.dev/blog/flet-mobile-strategy/#standalone-mobile-package-for-flet-app), Flet does not enable creating stand-alone mobile apps, but it is on the road-map. I guess Flutter with dart does that, and just now that I actually started to look at Flet that I found out that it is a server based mobile app.
That said, may I ask if is there an estimated time for this to be implemented?

**Comment by FeodorFitsner** [comment]
Flet does allow creating of standalone apps. See this example and guide: https://github.com/flet-dev/serious-python#usage
We are working on updated guidelines/packaging.
